eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -134,4 +136,12 @@
 
 <!--  Storage still depends on 1.0.0-preview.2 azure core. suppress this rule for now until storage has upgraded to preview-3 azure core-->
   <suppress checks="com.azure.tools.checkstyle.checks.ServiceClientBuilderCheck" files=".*[/\\]storage[/\\].*\.java"/>
+
+  <!-- ClientLogger class suppresion -->
+  <suppress checks="com.azure.tools.checkstyle.checks.GoodLoggingCheck" files=".*ClientLogger.java"/>
+  <!--  Azure Core Test -->
+  <suppress checks="com.azure.tools.checkstyle.checks.GoodLoggingCheck" files=".*[/\\]core[/\\]test[/\\].*\.java"/>
+  <!-- Already Public API-->
+  <suppress checks="com.azure.tools.checkstyle.checks.GoodLoggingCheck" files=".*CertificateUtil.java"/>
```

Comment:
This may be a little too broad. If you want to suppress an already public API class, make it very specific with fully-qualified classname because the suffix `*CertificateUtil` may be used in other places too.